buildkite/common

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+set -ex
+
+if [ "$BUILDKITE" = "true" ]; then
+    GIT_ID=${BUILDKITE_COMMIT:0:7}
+else
+    GIT_ID=$(git rev-parse --short=7 HEAD)
+fi
+
+if [ "$BUILDKITE" = "true" ]; then
+    GIT_BRANCH=$BUILDKITE_BRANCH
+else
+    GIT_BRANCH=$(git symbolic-ref --short HEAD)
+fi
+
+# Deal with dependabot tags which look like
+#
+#   dependabot/npm_and_yarn/app/lodash-4.17.19
+#
+# But docker does not like
+GIT_BRANCH=$(echo $GIT_BRANCH | sed 's;/;-;g')
+
+BUILDKITE_DOCKER_AUTH_PATH=/var/lib/buildkite-agent/.docker/config.json
+
+# Export env vars needed for running test dependencies
+export REGISTRY=vimc
+export MONTAGU_ORDERLY_SERVER_VERSION=$(<$HERE/../config/orderly_server_version)
```

Comment:
`$HERE` isn't defined, so this isn't pulling in the version as desired. From buildkite logs: 
`./buildkite/common: line 27: /../config/orderly_server_version: No such file or directory`